annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -90,7 +90,16 @@
             "splice_acceptor_variant", "splice_donor_variant", "splice_region_variant"));
 
     private Map<String, String> variantMap = new HashMap<String, String>();
-    private List<String> hgvspNullClassifications = new ArrayList<>();
+    private static List<String> hgvspNullClassifications = new ArrayList<>();
```

Comment:
since this can be an immutable list, it can be initialized like this instead:
List<String> strings = Arrays.asList("foo", "bar", "baz");
see  http://stackoverflow.com/questions/1005073/initialization-of-an-arraylist-in-one-line

If the list was longer, I might consider making it a hashset .. that way lookup into the list would be ~ order(1) rather than order(n). But the overhead of hashing is probably more costly when the list size is under 10